src/index.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import mongoose from 'mongoose';
+import bodyParser from 'body-parser';
+import MovieRoute from './routes/MovieRoute';
+
+const app = express();
+mongoose.Promise = global.Promise;
+mongoose.connect('mongodb://localhost/movie-list');
+
+app.use(bodyParser.json());
+app.use(MovieRoute);
+app.use(function (err, request, response) {
```

Comment:
Hey I know you were talking about an issue you had with this, can you push up your fix?